scripts/release/issue-list.sh

```diff
@@ -40,19 +41,41 @@ function milestone_issues()
   # Remove enclosing quotes on each line
   issue_list=`echo "$issue_list" | tr -d \"`
 
+  # Replace \ which is left over from above command with "(double quote) and suppress warning
+  issue_list=`echo "$issue_list" | tr '\' '"' 2> /dev/null`
+
   # Adjust the issue links
   issue_list=`echo "$issue_list" | sed -e s/api.github.com.repos/github.com/g`
-  echo "$issue_list"
+
+  bugs="# Bugs"
```

Comment:
We should change the order. Features first, then bugs and last tasks. Whether one wants to switch the last two is maybe arguable, but features are most important.